### DISCUSSION TITLE
No match despite providing MusicBrainzID through --search-id

### DISCUSSION THREAD

**Discussion by mjmilloy** [beetbox/beets#5332] [Q&A]
Hi! I have a multi-disc boxset composed of previously-released albums with an MBID. The files are complete but the importer reports "No matching release found" when invoked with --search-id and the MBID. Tips/tricks/etc.?

**Comment by arsaboo** [comment]
Please provide the logs with `beet -v`. Make sure that MBID is valid and the tracks match.
